pkg/agent/baker.go

```diff
@@ -395,7 +395,7 @@ func getContainerServiceFuncMap(config *NodeBootstrappingConfiguration) template
 			return datamodel.AzureADIdentitySystem
 		},
 		"GetPodInfraContainerSpec": func() string {
-			return cs.Properties.OrchestratorProfile.GetPodInfraContainerSpec()
+			return profile.KubernetesConfig.ContainerRuntimeConfig["SandboxImageUrl"]
```

Comment:
https://github.com/Azure/AgentBaker/pull/326/files#r490622417

Yi changed it to be passed in by RP